scenarios/sqs_flag_shop/terraform/ec2.tf

```diff
@@ -4,7 +4,7 @@ resource "aws_key_pair" "bob-ec2-key-pair" {
 }
 
 resource "aws_instance" "cg_flag_shop_server" {
-  ami                         = "ami-0fc5d935ebf8bc3bc"
+  ami                         = "ami-0a313d6098716f372"
```

Comment:
So this would be a data call through Terraform
https://registry.terraform.io/providers/hashicorp/aws/latest/docs/data-sources/ami